src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

```diff
@@ -27,24 +23,48 @@ With Azion, you can customize the time the cache remains in the edge through *ti
 
 ---
 
-## Cache keys
+## Cache status
 
-A cache key is an index entry for an object in Azion's edge nodes. When a request is made to an edge application for the first time, the edge forwards the request to the origin server and generates a cache for each object received at the edge. When the next request for the same resource is made to the edge application, if the resource requested matches one already found through the cache key, the request won't be forwarded to the origin.
+Azion provides a layer to cache content in the edge network, which stands between origin servers and the end user. When content is successfully cached at the edge, it receives a [cache status HIT](#status-definitions) and it can be directly delivered to your users from the edge node nearest to them, reducing the frequency of access to origin servers.
+
+In the event the requested content isn't available in cache, the Edge Caching module minimizes the impact by maintaining a keep-alive connection with the origin server whenever possible. This approach avoids the overhead of frequent TCP/IP handshakes. Regardless of the number of simultaneous requests made to Azion's edge nodes, each edge node only needs to request content from the origin once per MISS or other statuses.
```

Comment:
Não é minha vertical aqui (Build), mas que eu saiba o módulo de Edge Caching não tem nada a ver com "mantaining a keep-alive connection with the origin".

Caching é capacidade de entregar respostas direto da edge, sem precisar sequer conectar na origem. Quando o cache precisa ser ATUALIZADO, aí sim uma conexão é feita à origem, mas o keep-alive não tem nenhuma relação direta com a configuração de Edge Caching em si...

Talvez, querem citar aqui o controle de `Thundering Herd`, que na verdade é um controle que permite que múltiplas requests ao mesmo tempo para um mesmo objeto gerem apenas 1 conexão para a origem. Tem que validar com o PM esse parágrafo.

Reply:
@Vitor-Azion

Reply:
Eu adicionaria no texto as duas práticas:

"When the requested content is not available in the cache, Azion's Edge Caching module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a cache MISS. Additionally, we implement Thundering Herd control to ensure that, in the face of multiple simultaneous requests for the same non-cached object, only one connection is established with the origin server. This approach ensures that each of Azion's edge nodes requests content from the origin server only once per MISS or other non-HIT statuses, thereby optimizing network efficiency and reducing the load on the origin server."

Reply:
@bru-andrade